consul/config.sls

```diff
@@ -1,9 +1,7 @@
-{% from "consul/map.jinja" import consul with context %}
+{% from slspath+"/map.jinja" import consul with context %}
```

Comment:
Reading though the `file` execution module doc I stumbled at these examples:
```jinja
{%- from salt['file.dirname'](tpldir) + '/vars.jinja' import parent_vars %}
```
```jinja
{%- from salt['file.normpath'](tpldir + '/../vars.jinja') import parent_vars %}
```

I immediately thought it would be really good match for this case. Looks much more solid.
Although, this variable -- `tpldir` -- seems to be not documented at all... After reading the code a little bit I think it is a way better alternative to the `slspath` variable, because it should be available in the context of every Jinja template, and not limited to the SLS files.

With this we possibly could just replace only the `{% from ... import ... %}` line in all templates to achieve relative path resolution without attempting to workaround the limited usage scope of `slspath`.

@ChrisLundquist , could you please adapt the above examples and test if those would match your use case?